commit 7dba62b1f13f6cc861502f039558b3cb599b6bf3 (refs/changes/56/4556/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-20T16:38:10-07:00 (1 year, 3 months ago)
    
    TRITON-622 'stdin.write' in promptYesNo func in common.js breaks on node v10

diff --git a/lib/common.js b/lib/common.js
index d9bf137..9a6152a 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -301,9 +301,9 @@ function promptYesNo(opts_, cb) {
     stdin.on('data', onData);
 
     function postInput() {
+        stdout.write('\n');
         stdin.setRawMode(false);
         stdin.pause();
-        stdin.write('\n');
         stdin.removeListener('data', onData);
     }
 
